internal/terraform/json.go

```diff
@@ -220,20 +214,20 @@ func ReadProviderConfigValues(data []byte, kind, name string) (map[string]interf
 	return result, nil
 }
 
-func resourceProviderConfig(kind string, name string, plan *plan) (ProviderConfig, error) {
-	// Find the provider_config_key for this resource.
+func resourceProviderConfig(kind string, name string, plan *plan) ProviderConfig {
```

Comment:
optional: It may be a cleaner interface to return (pc *ProviderConfig, ok bool) where ok represents whether the config was found.

Reply:
Yes, that's cleaner, thank you.